lib/tuya.js

```diff
@@ -581,6 +581,26 @@ const dataPoints = {
     // Moes switch with optional neutral
     moesSwitchPowerOnBehavior: 14,
     moesSwitchIndicateLight: 15,
+    // X5H thermostat
```

Comment:
Some of these datapoints are already defined for other devices.
Should I use predefined datapoints or create new ones for my device?
Is it possible some of those devices may receive a firmware update with updated datapoints and this will break my device?